src/imitation/scripts/multi_expert_demos.py

```diff
@@ -0,0 +1,150 @@
+"""Multiprocessed version of expert_demos."""
+from collections import OrderedDict
+import csv
+import multiprocessing.dummy
+import os.path as osp
+from typing import Iterable, List, Optional, Tuple
+
+from sacred.observers import FileStorageObserver
+import tensorflow as tf
+
+from imitation.scripts.config.multi_expert_demos import multi_expert_demos_ex
+from imitation.scripts.expert_demos import expert_demos_ex
+import imitation.util.multi as multi_util
+
+
+@multi_util.args_single_to_star
+def _job(
+  row: OrderedDict,
+  log_dir: str,
+  extra_named_configs: Iterable[str],
+  extra_config_updates: dict,
+) -> Tuple[OrderedDict, dict]:
+  """Start a `expert_demos_ex` run.
+
+  Params:
+    row: A CSV row encoding config settings for this run. Its fields should
+      include 'env_config' (str), 'reward_path' (str), 'reward_type' (str),
```

Comment:
This function looks for keys `"transfer_reward_path"`and `"transfer_reward_type"` not `"reward_path"` and `"reward_type"`.